fastly/user.go

```diff
@@ -0,0 +1,193 @@
+package fastly
+
+import (
+	"fmt"
+	"sort"
+	"time"
+)
+
+// User represents a user of the Fastly API and web interface.
+type User struct {
+	CreatedAt              *time.Time `mapstructure:"created_at"`
+	CustomerID             string     `mapstructure:"customer_id"`
+	DeletedAt              *time.Time `mapstructure:"deleted_at"`
+	EmailHash              string     `mapstructure:"email_hash"`
+	ID                     string     `mapstructure:"id"`
+	LimitServices          bool       `mapstructure:"limit_services"`
+	Locked                 bool       `mapstructure:"locked"`
+	Login                  string     `mapstructure:"login"`
+	Name                   string     `mapstructure:"name"`
+	RequireNewPassword     bool       `mapstructure:"require_new_password"`
+	Role                   string     `mapstructure:"role"`
+	TwoFactorAuthEnabled   bool       `mapstructure:"two_factor_auth_enabled"`
+	TwoFactorSetupRequired bool       `mapstructure:"two_factor_setup_required"`
+	UpdatedAt              *time.Time `mapstructure:"updated_at"`
+}
```

Comment:
Whilst I appreciate that you've sorted the field keys alphabetically, I'd prefer it if these followed the same convention as other structs in the client, such as the important field like `ID`, `Name`, `Role` coming first and the metadata `CreatedAt`, `UpdatedAt` and `DeletedAt` coming last.

Reply:
re-ordered in 02f2f2b ... better?